lib/tasks/database_updates/backfill_publisher_selected_wallet.rake

```diff
@@ -0,0 +1,105 @@
+namespace :database_updates do
+  desc 'Backfill Publisher Selected Wallets'
+  task :backfill_publisher_selected_wallets => :environment do
+    def handle_gemini
+      now = Time.zone.now
+      gemini_to_set = Publisher.
+        includes([:gemini_connection]).
+        joins(:gemini_connection).
+        where('gemini_connections.is_verified = TRUE').
+        where('publishers.email IS NOT NULL').
+        where('publishers.selected_wallet_provider_id IS NULL').
+        pluck(:id, 'gemini_connections.id') # use pluck to avoid loading user_authentication_token
```

Comment:
@yachtcaptain23 Note this is optimized for speed. Pluck avoids loading the associated token, and below with Uphold we chunk the work.

Reply:
Note we also use upsert_all to avoid updating one at a time. One downside of that is that the timestamps are reset, but the speed is worth it.